source/developers/how-tos/enable-python-upgrade-automation.rst

```diff
@@ -0,0 +1,44 @@
+How To Enable Python Upgrade Automation
+#######################################
+
+This How-to will walk you through the process of getting the automated python
+requirements upgrade jobs setup on a repository.
+
+Assumptions
+***********
+
+* You know how to use Git and GitHub PR Workflows
+
+* You have ``admin`` access to the repository.
+
+Steps
+*****
+
+1. Replace ``<repo_name>`` with the name of your repository in the following URL
+   and go to it:
+
+   .. code::
+
+      https://github.com/openedx/<repo_name>/actions/new?category=owner
```

Comment:
Why are we using the GitHub UI for this, instead of telling them to create a file the usual way>

Reply:
OK, let's try it like this.  My feeling when I've had GitHub pop open an editor for me to add a file (this just happened for dependabot.yml), is that I'm now not quite sure what will happen. Also, the later instruction say to check that the file is a correct copy, which will require them to go find where it is in our templates.